src/fundus/publishers/tr/__init__.py

```diff
@@ -0,0 +1,16 @@
+from fundus.publishers.base_objects import PublisherEnum, PublisherSpec
+from fundus.scraping.url import NewsMap, Sitemap
+
+from .haberturk import HaberturkParser
+
+
+class TR(PublisherEnum):
+    Haberturk = PublisherSpec(
+        name="Haberturk",
+        domain="https://www.haberturk.com/",
+        sources=[
+            Sitemap("https://www.haberturk.com/sitemap.xml"),
```

Comment:
There seems to be some stuff indexed in the sitemaps not referring to articles, like `sitemap_headlines`. Could make use of the `sitemap_filter` parameter of `Sitemap` to access only sitemaps referring articles. You can import everything you need from `fundus.scraping.filter`. You can find examples of how to use those in `fundus.publishers.us.__init__.py`. For further information read [this tutorial](https://github.com/flairNLP/fundus/blob/master/docs/4_how_to_filter_articles.md#url-filter).

Reply:
Regex filter added